proposals/gc/MVP.md

```diff
@@ -522,7 +538,7 @@ The opcode for heap types is encoded as an `s33`.
 | 0xfb14 | `array.get_s $t` | `$t : typeidx` |
 | 0xfb15 | `array.get_u $t` | `$t : typeidx` |
 | 0xfb16 | `array.set $t` | `$t : typeidx` |
-| 0xfb17 | `array.len $t` | `$t : typeidx` |
+| 0xfb17 | `array.len` | `u32` (TODO: remove, was typeidx) |
```

Comment:
Shouldn't this be
 | 0xfb17 | `array.len $t` | `$t : u32` |

Reply:
Well, the index has been removed from the abstract syntax, so the parameter is merely a detail of the binary format. Changed to `_ : u32`.